CHANGELOG.md

```diff
@@ -6,9 +6,142 @@
 
 ## v4.1
 
-### 4.1.0 - Highlights
+### Custom CSS
 
-#### Moved plugins
+Jupyterlab now supports automatic loading of custom CSS.
+Themes are the recommended method for customising the JupyterLab look and feel,
+while custom CSS is intended for minor personal adjustments.
+
+To opt-in start jupyterlab with `--custom-css` flag.
+The location for the `custom.css` is documented the section on [customizing the user interface](https://jupyterlab.readthedocs.io/en/latest/user/interface_customization.html#custom-css).
+
+```{note}
+`custom.css` is also available in Jupyter Notebook;
+while similar selectors can be used for components shared between JupyterLab 4.1 and Jupyter Notebook 7.0,
+a different set of selectors was used in the Notebook 6 and older.
+Users with the custom CSS file using the selectors from legacy Notebook versions
+will need to update them and note that tutorials may refer to the selectors for the old Notebook versions.
```

Comment:
Maybe we can leave a note to encourage users to open a post on Discourse if they have trouble updating?

Reply:
We can also just leave it like this and see if users open issues or not.